Library/Homebrew/keg_relocate.rb

```diff
@@ -1,4 +1,12 @@
+require "os/mac/relocate"
```

Comment:
Right now the pertinent code is in `os/mac/{ruby_,cctools,,}relocate`, does renaming all of these to "keg" instead of "relocate" and correcting the `require`s sound good to you?